code/modules/surgery/organs/ears.dm

```diff
@@ -54,8 +54,11 @@
 		REMOVE_TRAIT(owner, TRAIT_DEAF, EAR_DAMAGE)
 
 /obj/item/organ/ears/proc/adjustEarDamage(ddmg, ddeaf)
-	damage = max(damage + (ddmg*damage_multiplier), 0)
-	deaf = max(deaf + (ddeaf*damage_multiplier), 0)
+	if(owner.status_flags & GODMODE)
```

Comment:
Change this to an early-return
```dm
if (bla)
    return

// the rest of your code goes here, no else
```

Reply:
You also shouldn't return FALSE unless you also return TRUE in success cases.

Reply:
It didn't work properly without the else